src/homepage/_posts/2017-10-02-BlogPostInstallingJames.markdown

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "Blog post: Easy and secure James installation"
+date:   2017-10-03 16:13:22 +0200
+categories: james update
+---
+
+In a recent [blog post], Thibaut explain us how to easily set up a James sever on a personal domain.
```

Comment:
without a link to a profile, nobody knows we is Thibaut, right ?

Reply:
"explain" -> "explains"
"sever" -> "server"